src/Ninja/G1CP/Content/Fixes/Session/fix129_DrakeBodySkin.d

```diff
@@ -0,0 +1,45 @@
+/*
+ * #129 Drake has wrong body skin color
+ */
+func int G1CP_129_DrakeBodySkin() {
+    var int applied; applied = FALSE;
+
+    // Get necessary symbol indices
+    var int symbId; symbId = G1CP_GetNpcInstID("Grd_260_Drake");
+    if (symbId == -1) {
+        return FALSE;
+    };
+
+    // Find call to "Mdl_SetVisualBody"
+    const int bytes[2] = {zPAR_TOK_CALLEXTERN<<24, -1};
+    bytes[1] = MEM_GetFuncID(Mdl_SetVisualBody);
+    var int matches; matches = G1CP_FindInFunc(symbId, _@(bytes)+3, 5);
```

Comment:
Thanks to more recent changes we can replace these three lines by just one:
```D
// Find call to "Mdl_SetVisualBody"
var int matches; matches = G1CP_FindCall(symbId, 0, MEM_GetFuncID(Mdl_SetVisualBody));
```
It may be even possible to omit the call to `MEM_GetFuncID` - that should be tested.